src/devices/tuya.ts

```diff
@@ -13146,6 +13146,102 @@ const definitions: DefinitionWithExtend[] = [
+                .withUnit('s'),
+            e.illuminance(),
+            e
+                .numeric('far_mov_sensitivity', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(10)
+                .withValueStep(1)
+                .withDescription('the moving detecting sensitivity 1 meter away'),
+            e
+                .numeric('near_mov_sensitivity', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(10)
+                .withValueStep(1)
+                .withDescription('the moving detecting sensitivity  within 1 meter'),
+            e
+                .numeric('near_occ_sensitivity', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(10)
+                .withValueStep(1)
+                .withDescription('the presence detecting sensitivity  within 1 meter'),
+            e
+                .numeric('far_occ_sensitivity', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(10)
+                .withValueStep(1)
+                .withDescription('the presence detecting sensitivity  1 meter away'),
+            e
+                .numeric('near_detection', ea.STATE_SET)
+                .withValueMin(0.1)
+                .withValueMax(7)
+                .withValueStep(0.1)
+                .withDescription('The closest distance that can be detected')
+                .withUnit('m'),
+            e
+                .numeric('far_detection', ea.STATE_SET)
+                .withValueMin(0.1)
+                .withValueMax(7)
+                .withValueStep(0.1)
+                .withDescription('The farest distance that can be detected (moving)')
+                .withUnit('m'),
+            e
+                .numeric('occ_detection', ea.STATE_SET)
+                .withValueMin(0.1)
+                .withValueMax(7)
+                .withValueStep(0.1)
+                .withDescription('The farest distance that can be detected (present)')
+                .withUnit('m'),
+            e.binary('restore_factory', ea.STATE_SET, 'ON', 'OFF').withDescription('restore_factory'),
+            e.binary('led_indicator', ea.STATE_SET, 'ON', 'OFF').withDescription('turn on or off the led '),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [104, 'presence', tuya.valueConverter.trueFalse1],
+                [112, 'occupancy', tuya.valueConverter.trueFalseInvert],
+                [9, 'target_dis_closest', tuya.valueConverter.divideBy100],
+                [101, 'mov_delay', tuya.valueConverter.raw],
+                [102, 'none_delay', tuya.valueConverter.raw],
+                [103, 'illuminance', tuya.valueConverter.divideBy10],
+                [105, 'far_mov_sensitivity', tuya.valueConverter.raw],
+                [110, 'near_mov_sensitivity', tuya.valueConverter.raw],
+                [109, 'near_occ_sensitivity', tuya.valueConverter.raw],
```

Comment:
```suggestion
                [109, 'near_occupancy_sensitivity', tuya.valueConverter.raw],
```

Reply:
I left the keys in place as specified by the vendor (simple excel file with the datapoint specifications). Do you prefer readability or would you recommend to stick closer to the vendor documentation?